src/test/HostActivationTests/BundledAppWithSubDirs.cs

```diff
@@ -18,59 +18,100 @@ public BundledAppWithSubDirs(SharedTestState fixture)
             sharedTestState = fixture;
         }
 
-        [Fact]
-        private void Bundle_And_Run_App_With_Subdirs_Succeeds()
+        private static string GetHostName(TestProjectFixture fixture)
         {
-            var fixture = sharedTestState.TestFixture.Copy();
-            var hostName = Path.GetFileName(fixture.TestProject.AppExe);
+            return Path.GetFileName(fixture.TestProject.AppExe);
+        }
 
-            // Bundle to a single-file
-            // This step should be removed in favor of publishing with /p:PublishSingleFile=true
-            // once associated changes in SDK repo are checked in.
-            string singleFileDir = Path.Combine(fixture.TestProject.ProjectDirectory, "oneExe");
-            Directory.CreateDirectory(singleFileDir);
-            var bundler = new Microsoft.NET.HostModel.Bundle.Bundler(hostName, singleFileDir);
-            string singleFile = bundler.GenerateBundle(fixture.TestProject.OutputDirectory);
+        private static string GetPublishPath(TestProjectFixture fixture)
+        {
+            return Path.Combine(fixture.TestProject.ProjectDirectory, "publish");
+        }
 
-            // Run the bundled app (extract files)
-            Command.Create(singleFile)
+        private static DirectoryInfo GetBundleDir(TestProjectFixture fixture)
```

Comment:
Maybe move these static functions into a helper class instead of duplicating them in each test source file.

Reply:
I thought about that -- but these are auxiliary one-line helpers. Plus there was no good place to put the helpers because: 
* They depend on the specific way in which files are published, bundled, or extracted (directory names and such). So, it is not a good idea to add them to TestUtilities generically.
* They are used both by HostActivationTests and HostModel test.

So I let them be in the individual tests. I'm happy to extract the code out to a helper if we feel strongly about it.

Reply:
If you move the tests into a sub-namespace, then adding a bundle only helpers should be trivial.
Alternatively you can have a base class for the bundle tests and share that way.

I agree that moving these to TestUtils is probably an overkill, but currently there's already duplication in HostActivationTests alone.

Reply:
I Created a BundleTest test directory grouping which contains
- Microsoft.NET.HostModel.Bundle tests
- AppHost Bundler tests
- BundleHelpers -- containing common helper methods for the above two tests.